dashboard/app/helpers/pd/workshop_survey_results_helper.rb

```diff
@@ -166,33 +195,57 @@ def summarize_workshop_surveys(workshops:, facilitator_name_filter: nil, facilit
   def generate_workshop_daily_session_summary(workshop)
     summary = {
       this_workshop: {},
+      course_name: workshop.course
     }
 
     questions = get_questions_for_forms(workshop)
 
-    summary['questions'] = questions
+    summary[:questions] = questions
+
+    summary[:this_workshop] = generate_workshops_survey_summary([workshop], questions)
+
+    related_workshops =
+      if current_user.permission?(UserPermission::WORKSHOP_ADMIN)
+        [workshop]
+      elsif current_user.permission?(UserPermission::PROGRAM_MANAGER)
+        Pd::Workshop.where(regional_partner: current_user.regional_partners, course: workshop.course)
+      elsif current_user.permission?(UserPermission::WORKSHOP_ORGANIZER)
+        Pd::Workshop.organized_by(current_user).where(course: workshop.course)
+      else
+        Pd::Workshop.facilitated_by(current_user).where(course: workshop.course)
+      end
+
+    # Need to add start date filter
```

Comment:
Do we need a start date filter still, and is this work tracked somewhere besides this comment?

Reply:
Now implemented